ide/machine_setup.rst

```diff
@@ -154,6 +154,9 @@ Add aliases to ``~/.bash_aliases`` (adjust versions as needed)::
 This will allow you to run ``mvn`` using an explicit java version. Examples::
 
     # The two links are helpful for building without tests
+
+    TODO: ref:``
```

Comment:
Can you convert the link below to `` :refs`…` ``? Thanks.

Otherwise, I think things are ready to be merged.